### PR TITLE
Feat: analytics events update

### PR DESCRIPTION
**Pull request by sorodrigo**
Adds new events according to https://docs.google.com/spreadsheets/d/1TaZ3IN_Hoj9DccxX2opbB4uOMFEQ2VNcIe4ksn5B1Co/edit?usp=sharing

### CODE REVIEW

File: frontend/scripts/react-components/dashboard-element/dashboard-widget/dashboard-widget.component.jsx [hunk 1]
```diff
@@ -24,6 +24,15 @@ function DashboardWidget(props) {
 
   const [isModalOpen, openModal] = useState(false);
 
+  const openTableModal = useCallback(() => {
+    openModal(true);
+    openTableView(title);
```
Comment (Bluesmile82) [inline frontend/scripts/react-components/dashboard-element/dashboard-widget/dashboard-widget.component.jsx]: Shouldn't we specify that this is just for analytics?

File: frontend/scripts/analytics/profile-node.events.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      // eslint-disable-next-line
+      return `${data?.jurisdiction_name} - ${data?.column_name} - ${context?.countryName} - ${
+        context.commodityName
+      } - ${query?.year}`;
+    },
+    shouldSend: action => action.payload.endpoint === GET_NODE_SUMMARY_URL
+  },
+  {
+    type: PROFILE_NODE__ON_PDF_DOWNLOAD,
+    action: 'Download PDF',
```
Comment (Bluesmile82) [inline frontend/scripts/analytics/profile-node.events.js]: Could we add the PROFILE category? I'm not quite sure what is that [WHERE?] on the spreadsheet

File: frontend/scripts/analytics/profile-node.events.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    getPayload(action, state) {
+      const { data } = action.payload;
+      const {
+        location: { query },
+        app: { contexts }
+      } = state;
+      const context = contexts.find(ctx => Number(query.contextId) === ctx.id);
+
+      // eslint-disable-next-line
+      return `${data?.jurisdiction_name} - ${data?.column_name} - ${context?.countryName} - ${
```
Comment (Bluesmile82) [inline frontend/scripts/analytics/profile-node.events.js]: Jurisdiction name seems to be undefined for http://localhost:8081/profile-actor?nodeId=24603&contextId=1&year=2017
{hitType: "event", eventCategory: "profile", eventAction: "Profile page visited", eventLabel: "undefined - IMPORTER - BRAZIL - SOY - 2017"}

Comment (sorodrigo) [reply frontend/scripts/analytics/profile-node.events.js]: woah thanks nice catch!

File: frontend/scripts/analytics/profile-node.events.js (hunk 2 above)
Comment (sorodrigo) [reply frontend/scripts/analytics/profile-node.events.js]: yes I should have added that

File: frontend/scripts/react-components/dashboard-element/dashboard-widget/dashboard-widget.component.jsx (hunk 1 above)
Comment (sorodrigo) [reply frontend/scripts/react-components/dashboard-element/dashboard-widget/dashboard-widget.component.jsx]: will do

File: frontend/scripts/analytics/home.events.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  {
+    type: HOME__CLICK_ENTRYPOINT,
+    category: 'homepage',
+    action: 'Click on entry points',
+    getPayload(action) {
+      const route = action.payload;
+      if (route.type !== 'tool') {
+        return route.type === 'profileRoot' ? 'profiles' : route.type;
+      }
+      if (route.payload?.query?.state?.isMapVisible) {
```
[the remark this answers is not in the document]
Comment (sorodrigo) [reply frontend/scripts/analytics/home.events.js]: yeah but what the hell

[the remark this answers is not in the document]
Comment (Bluesmile82) [reply frontend/scripts/analytics/home.events.js]: You also have get from lodash: https://lodash.com/docs/4.17.11#get

[the remark this answers is not in the document]
Comment (sorodrigo) [reply frontend/scripts/analytics/home.events.js]: true, but we can take advantage of this in many places.

[the remark this answers is not in the document]
Comment (Bluesmile82) [reply frontend/scripts/analytics/home.events.js]: As long as it works in all browsers is good for me

[the remark this answers is not in the document]
Comment (sorodrigo) [reply frontend/scripts/analytics/home.events.js]: yes babel should transpile it down to `route.payload && route.payload.query && ...`

**Pull request merged**